python/swe/dockerfiles/create_index.py

```diff
@@ -11,6 +11,8 @@
 from composio.utils.logging import WithLogger
 
 
+print("This is a Test PR")
```

Comment:
The added print statement appears to be for testing purposes. Could you please clarify the intention behind this change? Is it meant to be temporary or part of a larger feature?

Additionally, it's generally not recommended to add print statements in production code unless they serve a specific debugging purpose. Consider using a logging framework instead if this is intended for debugging or informational purposes. The `WithLogger` class imported on line 11 could be utilized for this purpose.